_data/settings.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function () {
+  let siteRoot = process.env.SITE_ROOT;
+
+  return {
+    siteRoot,
```

Comment:
this will not be needed `/` would do the work I think.

Reply:
Okay